src/Aspire.Dashboard/wwwroot/js/app.js

```diff
@@ -53,15 +53,33 @@ function isScrolledToBottom(container) {
 }
 
 window.copyTextToClipboard = function (id, text, precopy, postcopy) {
-    let tooltipDiv = document.querySelector('fluent-tooltip[anchor="' + id + '"]').children[0];
+    let button = document.querySelector(`#${id}`);
```

Comment:
nit: `const` ? (same for the other three new `let`s below). I don't know if we have a JS style guide, but MDN and others I'm used to prefer `const`. I realize the original code had `let` too, I must have missed that at the time.